cmd/tsconnect/wasm/wasm_js.go

```diff
@@ -385,6 +391,7 @@ func (s *jsSSHSession) Run() {
 		User:            s.username,
 	}
 
+	reportProgress("Starting SSH client…")
```

Comment:
if you want to do more, on line 390 just above there's the HostKeyCallback you can do to show that you've gotten data from the SSH connection and it's doing the phase of the SSH protocol before auth, exchanging host keys.

Reply:
Good idea, done.